ebill_paynet/models/res_partner_bank.py

```diff
@@ -0,0 +1,30 @@
+from odoo import models
+
+
+class ResPartnerBank(models.Model):
+    _inherit = "res.partner.bank"
+
+    def build_qr_code_base64(
+        self,
+        amount,
+        free_communication,
+        structured_communication,
+        currency,
+        debtor_partner,
+        qr_method=None,
+        silent_errors=True,
+    ):
+        # FIXME Workaround to bypass error None type of native update
```

Comment:
Thanks for the details  :+1: 
I wonder why we have this issue with this module. And if the qr code generation works sometimes.

Reply:
Hi @TDu , there is a check of eligible qr [here](https://github.com/odoo/odoo/blob/bc8e2d67b5980bb963f0e663f3c3d761ca4af169/addons/account/models/res_bank.py#L37). In the native module `l10n_ch`, it has overrided this func [here](https://github.com/odoo/odoo/blob/bc8e2d67b5980bb963f0e663f3c3d761ca4af169/addons/l10n_ch/models/res_bank.py#L289), but it just returns True for a specific case